src/align.py

```diff
@@ -1,7 +1,8 @@
+from typing import Tuple
 """A module for translating between alignments and edits sequences."""
 
 
-def get_edits(p: str, q: str) -> tuple[str, str, str]:
+def get_edits(p: str, q: str) -> Tuple[str, str, str]:
```

Comment:
Update your Python if you need to import `Tuple`. The `Tuple`, `List`, etc. type objects were deprecated in Python 3.9.0 and will be removed from the language with the first release after October 8 2025 (five years are the release of Python 3.9.0). Yes, there are three+ more years until it becomes an issue, but you might as well get used to using the lower-case versions. In any case, you must be using Python 3.8 or lower, and there are nifty features in Python 3.9 and 3.10 that you should get to know anyway.

(and my code examples sometimes use Python 3.10, so you need at least that to run those).